code/modules/mob/mob_say.dm

```diff
@@ -124,6 +124,8 @@
 			mods[MODE_SING] = TRUE
 		else if(key == ";" && !mods[MODE_HEADSET] && stat == CONSCIOUS)
 			mods[MODE_HEADSET] = TRUE
+		else if(key == ";" && !mods[MODE_HEADSET] && stat != CONSCIOUS)
+			chop_to = length(key) + 1
```

Comment:
This isn't strictly needed mind, chop_to is already set to 2, so as long as one of the else statements passes it'll go through properly.

It'd be more concise to do something like check for the key and if headset is set, then do the stat check, but that obscures the intent of the check in the first place, so perhaps not.

~~man my old code is shit~~

Reply:
All the conditions that are found here take into account only the channels of the departments and they have the removal of the channel prefix, but at the same time for the general channel there is no condition under which the semicolon will be removed when a person is in a critical condition, so this was done, of course it is possible not the best implementation